src/osemgrep/cli_login/Login_CLI.ml

```diff
@@ -0,0 +1,114 @@
+(* Provides the 'Arg', 'Cmd', 'Manpage', and 'Term' modules. *)
+open Cmdliner
+
+(*****************************************************************************)
+(* Prelude *)
+(*****************************************************************************)
+(*
+   'semgrep login' command-line arguments processing.
+*)
+
+(*****************************************************************************)
+(* Types *)
+(*****************************************************************************)
+type conf = { logging_level : Logs.level option } [@@deriving show]
+
+(*****************************************************************************)
+(* Helpers *)
+(*****************************************************************************)
+
+(*************************************************************************)
+(* Command-line flags *)
+(*************************************************************************)
+
+(* ------------------------------------------------------------------ *)
+(* "Verbosity options" (mutually exclusive) *)
+(* ------------------------------------------------------------------ *)
+(* alt: we could use Logs_cli.level(), but by defining our own flags
+ * we can give better ~doc:. We lose the --verbosity=Level though.
+ *)
+let o_quiet : bool Term.t =
```

Comment:
We could probably factorize code with Scan_CLI.ml by moving this in CLI_common.ml

Reply:
We could also move it in Cmdline_helpers.ml, but the o_verbose doc message seems quite specific to Semgrep,
so maybe better in CLI_common.ml

Reply:
I initially hated cmdliner, but I now really like its compostionality! You can compose cmdline args! A bit like dune that is also super compositional; just move your submodule in your repo and boom, it's working. Move around things, no problem still working.

Reply:
agreed, done in [04b56c9](https://github.com/returntocorp/semgrep/pull/7637/commits/04b56c94045713e6539f46d873298cfb71162d21)